pkg/resource/deploy/step_generator.go

```diff
@@ -697,20 +698,8 @@ func (sg *stepGenerator) continueStepsFromRefresh(event ContinueResourceRefreshE
 	goal := event.Goal()
 	urn := event.URN()
 	old := event.Old()
-	invalid := event.Invalid()
 	aliasUrns := event.Aliases()
 
-	// Create the desired inputs from the goal state
-	inputs := goal.Properties
-	if old != nil {
-		// Set inputs back to their old values (if any) for any "ignored" properties
-		processedInputs, err := processIgnoreChanges(inputs, old.Inputs, goal.IgnoreChanges)
```

Comment:
Again, be good to get a bit of context on this deletion?

Reply:
Its just moved